smartsim/experiment.py

```diff
@@ -797,6 +796,89 @@ def reconnect_orchestrator(self, checkpoint: str) -> Orchestrator:
             logger.error(e)
             raise
 
+    def preview(
+        self,
+        output_format: t.Optional[t.Literal["html"]] = None,
+        output_filename: t.Optional[str] = None,
+        verbosity_level: t.Literal["info", "debug", "developer"] = "info",
+    ) -> None:
+        """Preview entity information prior to launch. This method
+        aggregates multiple pieces of information to give users insight
+        into what and how entities will be launched.  Any instance of
+        ``Model``, ``Ensemble``, or ``Orchestrator`` created by the
+        Experiment can be passed as an argument to the preview method.
+        :param output_format: Set output destination. The possible accepted
+        output formats are `json`, `xml`, `html`, `plain_text`, `color_text`.
+        A filename is required if an output format is specified. If no output
+        format is set, the preview will be output to stdout. Defaults to None.
+        :type output_type: str
+        :param output_filename: Specify name of path to write preview data to.
+        Only needed when an output format has been specified. Defaults to None.
+        :type output_filename: str
+        :param verbosity level: the verbosity level.
+        info: Display User defined fields and entities
+        debug: Display user defined field and entities and auto generated
+        fields.
+        developer: Display user defined field and entities, auto generated
+        fields, and run commands.
+        Defaults to info.
+        :type verbosity_level: str
+        """
+
+        rendered_preview = previewrenderer.render(self, verbosity_level)
+
+        self._check_verbosity_level(verbosity_level)
+
+        if output_format:
```

Comment:
does this need to be if/else? can self._check_output_format accept None and just do 
```py
if not output_format_arg:
  return
```